service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Party.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.Address;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+public class Party {
+    public final String partyID;
+    public final String partyType;
```

Comment:
Please crate enum for party type and use it here. If not other developers will build their code around string values.